lighthouse-core/audits/image-aspect-ratio.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * @license Copyright 2017 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+ /**
+  * @fileoverview Checks to see if the aspect ratio of the images used on
+  *   the page are equal to the aspect ratio of their display sizes. The
+  *   audit will list all images that don't match with their display size
+  *   aspect ratio.
+  */
+'use strict';
+
+const Audit = require('./audit');
+
+const URL = require('../lib/url-shim');
+
+class ImageAspectRatio extends Audit {
+  /**
+   * @return {!AuditMeta}
+   */
+  static get meta() {
+    return {
+      category: 'Images',
+      name: 'image-aspect-ratio',
+      description: 'Uses Images with appropriate aspect ratio',
```

Comment:
@kaycebasques any thoughts or guidance on descriptions and helpText here? I had thought we decided to move away from the "uses" since it can be confusing if there are no images at all, but I can't remember what we settled on.